### PR TITLE
Feature/us1309 corkboard listing

### PR DESCRIPTION
**Pull request by bzmillerboy**

### CODE REVIEW

File: crossroads.net/app/corkboard/corkboard-listings.html [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+        <p>
+          Posts are deleted after 30 days to keep things neat and tidy.
+        </p>
+      </div><!--/corkboard-intro-->
+
+      <div class="col-xs-12 col-sm-6 col-md-4 corkboard-browse">
+
+        <h4 class="flush-top">Browse</h4>
+
+          <a href="#" class="btn btn-gray block btn-labeled push-half-bottom">Needs Posted
```
Comment (saylormic) [inline crossroads.net/app/corkboard/corkboard-listings.html]: leaving # in href will cause problems with angular routing

File: crossroads.net/styles/modules/buttons.scss [hunk 2]
```diff
@@ -3,18 +3,69 @@
...
+    border-bottom-right-radius: $border-radius-base;
 
     .icon {
-      margin-top: 2px;
+      margin-top: 6px;
+      color: #fff;
+    }
+  }
+
+  &.btn-gray {
```
Comment (saylormic) [inline crossroads.net/styles/modules/buttons.scss]: I see this was really carried over from Milacron, but we should use the Bootstrap mixin for creating new buttons. 

Example:

.btn-default {
  @include button-variant($btn-default-color, $btn-default-bg, $btn-default-border);
}

**Comment by bzmillerboy** [comment]
Good catch, I've fixed both of these.

**Comment by danrye** [comment]
.DS_Store should be in the .gitignore and removed from the repository

**Comment by bzmillerboy** [comment]
Fixed DS_Store...well Dan and Matt B did for me

**Pull request merged**